ansible/roles/products/common/validation/tasks/main.yml

```diff
@@ -30,6 +30,14 @@
     quiet: yes
   when: playbook=="teleop_deploy"
 
+- name: Make sure arm not used in docker_deploy
+  assert:
+    that:
+     - "'arm_' not in product"
+    fail_msg: "You cannot use the docker_deploy playbook with an arm. This requires a NUC, please plug the arm into the nuc and run the server_and_nuc playbook."
+    quiet: yes
+  when: playbook=="docker_deploy"
```

Comment:
uppercase 2nd NUC. Don't put info about the name of a different playbook as we can use also teleop_deploy or future others

Reply:
"This playbook is not suited for arm-related products." (maybe?)